moped-editor/src/views/projects/signalProjectTable/SignalProjectTable.js

```diff
@@ -118,7 +118,7 @@ const SignalProjectTable = () => {
     project["project_types"] = project_types;
 
     // project sponsor
-    project["project_sponsor"] = entityList.find(
+    project["project_sponsor_object"] = entityList.find(
       e => e.entity_id === project?.project_sponsor
     );
```

Comment:
using the same variable name was the fatal flaw here. After adding the windowResize hook, the table was rerendering, but not querying the data. So it was trying to match this to an object instead of a number, which would then result in undefined/empty.

Reply:
🙃  Thanks for figuring this one out!